fury/utils.py

```diff
@@ -706,7 +706,7 @@ def get_actor_from_primitive(vertices, triangles, colors=None,
 
 
 def repeat_sources(centers, colors, active_scalars=1., directions=None,
-                   source=None, vertices=None, faces=None):
+                   source=None, vertices=None, faces=None, orientation=None):
```

Comment:
> Testing the orientation argument is not possible without a test on repeat_sources.

Yes, please do add a test in `fury.tests.test_utils.py`

Reply:
Your implementation of test_advanced_geometry_actor is nice, thank you. I've committed the change.

repeat_sources is used by several actors. Testing its functionality requires careful thought and definitely some effort. 
Because I did not author this routine, and am only trying to submit a bug fix to the code, I respectfully decline.

Reply:
I understand. You have addressed all my comments. IDK if @skoudoro has any more comments.